_shared/project/pyproject.toml

```diff
@@ -173,6 +173,25 @@ good-names = [
 output-format = "colorized"
 score = "no"
 
+[tool.mypy]
+warn_unused_configs = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+
+disable_error_code = [
+  {% if include_exists("mypy/disables") %}
+    {{- include("mypy/disables", indent=4) -}}
+  {% endif %}
+]
```

Comment:
Could put the whole `disable_error_code = [ ... ]` block inside the `{% if %}`, so that if there's no include file you don't get an empty `disable_error_code = []`.